md/tomcat-bundle.md

```diff
@@ -84,11 +80,42 @@ If this is your first installation, copy the file to the `<TOMCAT_HOME>/setup/pl
 If this is a license update, use [the *Platform setup tool*](BonitaBPM_platform_setup.md#update_platform_conf).
 
 
+### Change the default credentials (optional, recommended for production)
+
+As a security precaution, we **strongly recommend** that before you start your application server, you change the default username and password used for the platform administrator and for the default tenant administrator.
+
+#### Platform administrator
+
+The username and password for the platform administrator are defined in the file [`bonita-platform-community-custom.properties`](BonitaBPM_platform_setup.md), by the following properties:
+
+* `platformAdminUsername` defines the username (default `platformAdmin`)
+* `platformAdminPassword` defines the password (default `platform`)
+
+This password is used for platform-level administration tasks, such as creating a tenant.
+
+#### Tenant administrator
+
+Each tenant has an administrator, with a tenant-specific username and password. The tenant administrator is also known as the tenant technical user.
+
+When the platform is created, default values for the tenant administrator username and password are defined in the file [`bonita-tenant-community-custom.properties`](BonitaBPM_platform_setup.md), by the following properties:
+
+* `userName` defines the username (default `install`)
+* `userPassword` defines the password (default `install`)
+
+When you create a tenant, the tenant administrator is created with the default username and password, unless you specify new values. 
+Change these tenant-specific credentials for an existing tenant by updating the `userName` and `userPassword` properties in `bonita-tenant-community-custom.properties`.
+
+::: warning
+For the default tenant, the tenant administrator username and password must also be defined in file [`platform-tenant-config.properties`](BonitaBPM_platform_setup.md), with exactly the same values that you set in `bonita-tenant-community-custom.properties`. 
```

Comment:
Same remark here. Why do we have to change these properties in 2 files with the exact same values ?